gradle.properties

```diff
@@ -8,5 +8,8 @@
 org.gradle.jvmargs=-Xmx3g
 # enable Gradle build cache
 org.gradle.caching=true
-
+# enable Gradle configuration cache
+org.gradle.configuration-cache=true
+# enable parallel loading of the configuration cache
+org.gradle.configuration-cache.parallel=true
```

Comment:
Any idea how much time we can save with this?

Reply:
no... really hard to tell.. I don't think an awful lot but who knows..

Reply:
we might be able to do the same with node_modules for the frontend (if not done already)

Reply:
ah ok. No, I don't think something like this is done for the node_modules. We are using this Gradle NPM plugin so it might be a bit tricky, not sure.